substrate/frame/support/src/traits/tasks.rs

```diff
@@ -57,7 +57,7 @@ pub trait Task: Sized + FullCodec + TypeInfo + Clone + Debug + PartialEq + Eq {
 	/// Performs the work for this particular `Task` variant.
 	fn run(&self) -> Result<(), DispatchError>;
 
-	/// Returns the weight of executing this `Task`.
+	/// Returns the weight of `run` function.
```

Comment:
Initially I wanted to have method `weight_of_is_valid`, thus this change, but better to do in a different PR
```suggestion
	/// Returns the weight of executing this `Task`.
```